MODAK/src/MODAK.py

```diff
@@ -111,6 +123,45 @@ def get_opt_container_runtime(self, job_json_data):
             new_container = self._map.map_container(job_json_data)
         logging.info(f"Optimal container: {new_container}")
         return new_container if new_container else ""
+    
+    def get_buildjob(self, job_json_data):
+        logging.info('Creating build script for job')
+        logging.info('Processing build data' + str(job_json_data))
+
+        # patch in the build job as the job task
+        # First, figure out what we're patching in:
+        try:
+            buildsrc = job_json_data["job"]["application"]["build"]["src"]
+        except KeyError:
+            logging.info("No job.application.build.src section, returning empty response")
+            return ""
+        buildcmd = job_json_data["job"]["application"]["build"]["build_command"]
+        final_build = ""
+        if buildsrc[-4:] == ".git":
+            final_build = f"git clone {buildsrc}\n"
+        else:
+            final_build = f"wget --no-check-certificate {buildsrc}\n"
+        
+        process_per_node=1
+        try:
+            process_per_node = job_json_data["job"]["application"]["build"]["build_parallelism"]
+        except KeyError:
+            pass
+        final_build += re.sub(
+            pattern="\{\{[\s]*BUILD_PARALLELISM[\s]*\}\}",
```

Comment:
```suggestion
            pattern="\{\{\s*BUILD_PARALLELISM\s*\}\}",
```

Reply:
If you're pulling in jinja anyway, I'm going to just import it and have it handle this.